config/params.ini

```diff
@@ -89,6 +89,11 @@ id_items=1
 open_chests=1
 fill_shared_stash_first=0
 
+[gamble]
```

Comment:
tbh I think we could even simplify futher. Get rid of `[gamble]` and `gamble=0` and just put `gamble_items=` under `[general]` or something. then update logic of the functions to instead of saying `if gamble:` do `if gamble_items:` That way we only add one line to config. If you don't want to gamble just leave the `gamble_items` blank. I know aeon's been trying to trim the config files based on his most recent commits.